mask-amd.js

```diff
@@ -53,6 +53,9 @@ define('mask', ['doc'], function($) {
 		});
 
 		$el.on('blur', function(event) {
+			if ($el.val().trim() === '') {
```

Comment:
**suggestion: ** could create to variable for caches `$el.val()`?

@tcelestino